node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -589,3 +697,71 @@ func (o GetDmsPathHelperGeneric) WaitForDmToExist(volumeUuid string, volumeNguid
 	}
 	return "", err
 }
+
+func (o GetDmsPathHelperGeneric) GetDmObjectsByIdentifier(wantedDmObjects []string, outputToCheckIn string) map[string]bool {
```

Comment:
```suggestion
func (o GetDmsPathHelperGeneric) ParseFieldValuesOfIdentifiers(identifiers []string, mpathdOutput string) map[string]bool {
```
IMO the `ToCheckIn` part is inaccurate and is already implied by the parameter.
`Parse` is more accurate than `Get`, because the values are already in the given output.
`ByIdentifier` could imply the map keys are the identifiers, even though the map is used as a set of the field values.

Reply:
done

Reply:
I still think `ParseMpathNameOfVolumePath` is better than `GetMpathDeviceNameFromProcMounts`.
how about simply `ParseMpathName`? @ArbelNathan any preference?

Reply:
done, change to `ParseMpathName`

Reply:
actually, the output is what being parsed. maybe `ExtractMpathName` is more accurate?
sorry for the confusion.
@ArbelNathan your input?

Reply:
by looking at these lines:
```
volumeIds := o.Helper.ParseMpathName(volumeIdVariations, mpathdOutput)
```
```
dms := o.Helper.ParseMpathName(volumeIdVariations, mpathdOutput)
```
it seems `MpathName` is not relevant for the "generic" function name (maybe only for one of the caller's name).
so maybe `ParseDmFieldValues`?

@ArbelNathan help

Reply:
`parse` is indeed done on the output but it's not the return value, the values of the parsing are `volumeIdVariation` and `dmFieldValue`, where the return value is a `map` of `dmFieldValue:bool` so I don't think that `parse` should be in the name.
with that said, `get` is not relevant as well cause as @oriyarde said it's already in the given output.

I don't understand how `volumeIds` can be a product of this method, if it's a possible output the whole method should be refactored (the return value is clearly `dmFieldValues`)

I think the best offer here is `ExtractDmFieldValues`

Reply:
done